### ISSUE TITLE
Import failures 

### ISSUE THREAD

**Issue by graik** [strubelab/multiprot#2]
On a fresh checkout:
```sh
git clone ...
cd multiprot
python3 multiprot/testing.py -e long
```
This gives several import errors (all tests fail):
```py
ERROR:root:Import failure in builder: ModuleNotFoundError("No module named 'multiprot.builder'; 'multiprot' is not a package")
ERROR:root:Import failure in errors: ModuleNotFoundError("No module named 'multiprot.errors'; 'multiprot' is not a package")
ERROR:root:Import failure in examples_wrapper: ModuleNotFoundError("No module named 'multiprot.examples_wrapper'; 'multiprot' is not a package")
ERROR:root:Import failure in multiprot: ModuleNotFoundError("No module named 'multiprot.multiprot'; 'multiprot' is not a package")
ERROR:root:Import failure in pulchra: ModuleNotFoundError("No module named 'multiprot.pulchra'; 'multiprot' is not a package")
ERROR:root:Import failure in ranch: ModuleNotFoundError("No module named 'multiprot.ranch'; 'multiprot' is not a package")
ERROR:root:Import failure in testing: ModuleNotFoundError("No module named 'multiprot.testing'; 'multiprot' is not a package")
ERROR:root:Import failure in tests_multiprot: ModuleNotFoundError("No module named 'multiprot.tests_multiprot'; 'multiprot' is not a package")
```

**Comment by graik** [comment]
There are two problems at play, it seems:

  * relative imports don't work for the testing script. E.g.:
   `from errors import *` cannot always be resolved from outside the package

  * changing this to `from multiprot.errors import *` solves the first issue but creates another one: there is also a multiprot.py script in the multiprot package and, depending on the context, this module is loaded instead of the package

We have to rename the multiprot script to something else and use only absolute imports. That's now also recommended by most python coding guidelines.

**Comment by graik** [comment]
Addressed in pull request #3

**Comment by guzmanfj** [comment]
I did three things to solve this issue (with your help and Arnaud's of course):
1. Merged your branch with the fix, which changed all relative to absolute imports
2. Changed the name of the `multiprot.py` script again to `parseChains.py`, and it now will be a regular module and not the end user script (and updated import statements accordingly)
3. Created a NEW script (Arnaud's idea) in `multiprot/scripts/` called **multipr**, which will call the previously mentioned module and builder.py to build the model(s). 

I had to create this new script file because removing the .py extension from `parseChains.py` (previously `multiprot_script.py`, previously `multiprot.py`) to make it a cleaner executable script would prevent it from being tested, so now this `multipr` script is much shorter and calls only a few methods.
